contrib/opencensus-ext-django/opencensus/ext/django/middleware.py

```diff
@@ -46,8 +46,8 @@
 REQUEST_THREAD_LOCAL_KEY = 'django_request'
 SPAN_THREAD_LOCAL_KEY = 'django_span'
 
-BLACKLIST_PATHS = 'BLACKLIST_PATHS'
-BLACKLIST_HOSTNAMES = 'BLACKLIST_HOSTNAMES'
+EXCLUDELIST_PATHS = 'EXCLUDELIST_PATHS'
```

Comment:
If I understand correctly, this is a breaking change?
@aabmass please make sure that this is okay / we bump major release version and mention this in the changelog.

Reply:
The packages that are breaking will be flask, django and pyramid. Giving the current versioning strategy (we update the packages that change with the core Opencensus package), which is currently at v0.7.10, what should we update this to?

Reply:
If we bump the major version to v1.7.0, it wouldn't really make sense in the future if other packages change and we match them to a new major version. Should we not have packages be updated in lockstep?

Reply:
For semver major versions < 1, minor version bumps are not compatible: https://semver.org/#spec-item-4. So from semver perspective, we don't need to do a major version bump.

If users are using the "compatible release" specifier (or stricter) like `opencensus-foo ~= 0.7.0`, it will not automatically start installing the next minor version.